recipes/boost/all/conanfile.py

```diff
@@ -839,6 +819,25 @@ def package(self):
         if self.options.header_only:
             self.copy(pattern="*", dst="include/boost", src="%s/boost" % self._boost_dir)
 
+        if self.settings.os == "Emscripten":
+            self._create_emscripten_libs()
```

Comment:
I was expecting to have this done in the `build()` step instead of the `package()` one. Is it possible?

Reply:
By default what is being package are the .bc files. I think that is being done from the b2 and I think is triggered by the 

```
        flags.extend(["install",
                      "--prefix=%s" % self.package_folder,
                      "-j%s" % tools.cpu_count(),
                      "--abbreviate-paths",
                      "-d%s" % str(self.options.debug_level)])
```

So the bc files will always be included in the package folder (please correct me if I am wrong). So I think it makes more sense to do the transformation there, since it is the final place of the object files (Maybe we should also delete the original bc files -- but I am not 100% sure, since I have just started experimenting with wasm. The documentation was not so clear (https://emscripten.org/docs/compiling/Building-Projects.html#integrating-with-a-build-system) ). 

Theoretically, we might be able to do the transformation in the build folder, but we will have to replicate the packaging logic of b2. 

What do you think?

Reply:
thanks for the clarification. If those files are generated in the install step, then I am fine with that. @sse4 could you confirm this?

Reply:
yes, this is accurate

Reply:
I still think that for future iterations of the recipe, would be more clear to have this logic in `build()` without moving the b2 install from `package()` (if that is possible). Thank you!